### PR TITLE
Feature/api app

### PR DESCRIPTION
**Pull request by jdiouf**
Added APIs to get catalogs, get all experiences,, and edit metadata

### CODE REVIEW

File: app/api/urls.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+router = DefaultRouter()
+
+app_name = 'api'
+
+urlpatterns = [
+    path('catalogs/', views.catalogs,
+         name='catalogs'),
+    path('experiences/', views.list_experiences,
+         name='experiences'),
+    path('experiences/<str:course_id>/', views.experience,
```
Comment (mattdrayer) [inline app/api/urls.py]: can we call this `experience_id` instead of `course_id`?

File: app/api/views.py [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+                        status.HTTP_500_INTERNAL_SERVER_ERROR)
+    except Exception as err:
+        logger.error(err)
+
+        return Response(errorMsg,
+                        status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+
+@api_view(['GET', 'PATCH'])
+def experience(request, course_id):
```
Comment (mattdrayer) [inline app/api/views.py]: --> `experience_id`. (or just `id)`

**Review by mattdrayer — APPROVED** [review]
One suggestion for a parameter name, otherwise LGTM!

**Review by SHGODE — APPROVED** [review]
approve

**Pull request merged**